src/scripts/shell/dimAppCtrl.controller.js

```diff
@@ -132,19 +132,12 @@ function DimApp(ngDialog, $rootScope, loadingTracker, dimPlatformService, $inter
 
       var language = translate ? vm.settings.language + '/' : ''; // set language
 
-      var file;
-      try {
-        file = require('app/views/' + language + name + '.template.html');
-      } catch (e) {
-        file = require('app/views/en/' + name + '.template.html');
-      }
-
       if (result) {
         result.close();
       } else {
         ngDialog.closeAll();
         result = ngDialog.open({
-          template: file,
+          template: require('app/views/' + language + name + '.template.html'),
```

Comment:
Wow that's crazy. I'm looking at https://github.com/webpack/webpack/issues/118#issuecomment-28559053 for some details. Wouldn't have expected this to work.

I think this does mean that no matter what everything with `.template.html` extension in `views` will be included no matter what.